src/App.js

```diff
@@ -1,8 +1,9 @@
 import './App.css';
 import Header from './Header';
 import Navbar from './Navbar';
-import Card from './Card';
 import { useEffect, useState } from 'react';
+import { Routes, Route } from "react-router-dom";
+import HomePage from './pages/Homepage';
 
 function App() {
   const [character, setCharacter] = useState([]);
```

Comment:
The character variable is actually an array of characters. You should rename it to characters to avoid confusion.